### PR TITLE
feat: try to do version errors like the migrations

### PR DESCRIPTION
**Pull request by beckermr**
PR Checklist:

- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] if you are adding a new page under `docs/` or `community/`, you have added it to the sidebar in the corresponding `_sidebar.json` file
- [x] put any other relevant information below

### CODE REVIEW

File: src/components/StatusDashboard/styles.module.css [hunk 1]
```diff
@@ -237,7 +237,7 @@
   max-width: 100%;
 }
 
-.errored_item_content pre {
+.errored_item_content {
```
Comment (beckermr) [inline src/components/StatusDashboard/styles.module.css:240]:
```suggestion
.errored_item_content pre {
```

File: src/components/StatusDashboard/version_updates.jsx [hunk 2]
```diff
@@ -92,7 +92,7 @@ export default function VersionUpdates({ onLoad }) {
               </div>
               <div className={styles.errored_item_content}
                 style={{ display: !expanded[item] && "none" }}>
-                <pre>{errors[item]}</pre>
+                {errors[item]}
```
Comment (beckermr) [inline src/components/StatusDashboard/version_updates.jsx:95]:
```suggestion
                <pre dangerouslySetInnerHTML={{ __html: errors[item]}} />
```

**Comment by beckermr** [comment]
@jaimergp @HaudinFlorence Check out this version of the fix:

Screenshot 2024-07-18 at 12 43 03 PM

**Comment by jaimergp** [comment]
Hm, yes, this would work for now without changing the backend. Long-term I'd prefer the JSON based approach but that's more work 😬 So yes, LGTM.

**Comment by jaimergp** [comment]
Oh, nice, the unsolvable items have a different style:

image

image


Maybe we should make them more consistent (I like the unsolvable format better).

**Pull request merged**